tests/functional/cross_db_utils/fixtures.py

```diff
@@ -0,0 +1,115 @@
+macros__test_assert_equal_sql = """
+{% test assert_equal(model, actual, expected) %}
+select * from {{ model }} where {{ actual }} != {{ expected }}
+
+{% endtest %}
+"""
+
+
+# dateadd
```

Comment:
This file contains the "project" file contents, all defined as Python strings. These can also live right alongside the test cases, but it can be nice to keep the concerns separated.